properties/models.py

```diff
@@ -92,3 +92,14 @@ def get_absolute_url(self):
     def delete(self, *args, **kwargs):
         self.images.all().delete()
         super().delete(*args, **kwargs)
+
+
+class Document(BasedModel):
+    STATUS_CHOICES = (('pending', 'Pending'), ('approved', 'Approved'), ('rejected', 'Rejected'),)
```

Comment:
Does this formatting did by the djlint or some other editor? because that is not how tuples are formatted

Reply:
yes they should be like this 
STATUS_CHOICES = (
    ('pending', 'Pending'),
    ('approved', 'Approved'),
    ('rejected', 'Rejected'),
)
but i used djlint and it did not change choices like this. Kept them inline as they were.

Reply:
> Does this formatting did by the djlint or some other editor? because that is not how tuples are formatted

yes they should be like this
STATUS_CHOICES = (
('pending', 'Pending'),
('approved', 'Approved'),
('rejected', 'Rejected'),
)
but i used djlint and it did not change choices like this. Kept them inline as they were.